itou/www/apply/views/process_views.py

```diff
@@ -1021,3 +1028,95 @@ def add_or_modify_prior_action(request, job_application_id, prior_action_id=None
         "final_hr": prior_action is not None,
     }
     return render(request, "apply/includes/job_application_prior_action_form.html", context)
+
+
+@login_required
+@require_POST
+def rdv_insertion_invite(request, job_application_id):
+    try:
+        job_application = (
+            JobApplication.objects.is_active_company_member(request.user)
+            .select_related("job_seeker", "to_company")
+            .get(id=job_application_id)
+        )
+    except JobApplication.DoesNotExist:
+        return render(
+            request,
+            "apply/includes/buttons/rdv_insertion_invite.html",
+            {"job_application": None, "state": "error"},
+        )
+
+    # Ensure company has RDV-I configured
+    if not job_application.to_company.rdv_insertion_id:
+        return render(
+            request,
+            "apply/includes/buttons/rdv_insertion_invite.html",
+            {"job_application": None, "state": "error"},
+        )
+
+    invitation_request = InvitationRequest.objects.filter(
+        job_seeker=job_application.job_seeker,
+        company=job_application.to_company,
+        created_at__gt=timezone.now() - settings.RDV_INSERTION_INVITE_HOLD_DURATION,
+    ).first()
```

Comment:
```suggestion
    ).order_by("-created_at").first()
```
(J'ai l'impression qu'on pourrait avoir plusieurs `InvitationRequest` en DB ?)